src/current/v23.2/known-limitations.md

```diff
@@ -165,7 +138,17 @@ In cases where the partition definition includes a comparison with `NULL` and a
 
 {% include {{page.version.version}}/known-limitations/sql-cursors.md %}
 
-### `SELECT FOR UPDATE` locks are dropped on lease transfers  and range splits/merges
+### Limitations for `SELECT FOR UPDATE`
+
+- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) places locks on each key scanned by the base index scan. This means that even if some of those keys are later filtered out by a predicate which could not be pushed into the scan, they will still be locked.
+
+    [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/75457)
+
+- [`SELECT FOR UPDATE`]({% link {{ page.version.version }}/select-for-update.md %}) only places an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which aquires a lock on all indexes.
```

Comment:
I think this is true but could use a quick fact-check

```suggestion
- By default under `SERIALIZABLE` isolation, [`SELECT FOR UPDATE`and `SELECT FOR SHARE` ]({% link {{ page.version.version }}/select-for-update.md %}) only place an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which acquires a lock on all indexes.
```

Reply:
May need to update to instead reflect https://github.com/cockroachdb/cockroach/issues/114737#issuecomment-1819750886 and link to that issue. @michae2 could you please review this and suggest relevant KL text?

Reply:
@mikeCRL I propose we roughly keep this text, but qualify it. Something like:

> Under serializable isolation, `SELECT FOR UPDATE` only places an unreplicated lock on the index being scanned by the query. This diverges from PostgreSQL, which aquires a lock on all indexes. This limitation does not exist when using `SELECT FOR UPDATE` under read committed isolation or with `optimizer_use_lock_op_for_serializable` set to true when using `SELECT FOR UPDATE` under serializable isolation.

Also I do think we should change the tracking link to https://github.com/cockroachdb/cockroach/issues/114737

Reply:
Fixed, and using the wording that is currently published [here](https://github.com/cockroachdb/docs/blob/main/src/current/v23.2/known-limitations.md#limitations-for-for-update-and-for-share-locks-under-serializable). 

@michae2, the currently published doc states that `enable_durable_locking_for_serializable` is the cluster setting to use here, and `optimizer_use_lock_op_for_serializable` is associated with the "unreplicated lock on the index being scanned by the query" limitation. Does the published version need to be corrected?

Reply:
@taroface `enable_durable_locking_for_serializable` changes the unreplicated locks to replicated locks, and `optimizer_use_lock_op_for_serializable` changes the locking on the index being scanned to locking on the primary index.